### ISSUE TITLE
Create FIA6 database

### ISSUE THREAD

**Issue by mdietze** [PecanProject/pecan#899]
Need to update schemas and create downloadable copy of  FIA database v6. This should exist in parallel to FIA5, not blow it away, as it may take time to migrate all code to the new schema.

**Comment by mdietze** [comment]
In a recent PEcAn weekly call, @robkooper mentioned that in the future he wants to grab the FIA via the EcoDataRetriever rather than directly from FIA. See issue #741

**Comment by dlebauer** [comment]
Just spoke with @ethanwhite and the FIA use case for ecodataretriever came up ... he said that he and Jeremy are going to be working on updating their FIA data importer in the next year. 

@robkooper have you dealt with v6 in the past year or two?

**Comment by robkooper** [comment]
yes, biggest stopage for me is the initial download of the data which takes a few hours and at that point I have moved to another project/problem.
